lib/matplotlib/cbook.py

```diff
@@ -503,9 +505,8 @@ def get_sample_data(fname, asfileobj=True, *, np_load=False):
     the Matplotlib package.
 
     If the filename ends in .gz, the file is implicitly ungzipped.  If the
-    filename ends with .npy or .npz, *asfileobj* is True, and *np_load* is
-    True, the file is loaded with `numpy.load`.  *np_load* currently defaults
-    to False but will default to True in a future release.
+    filename ends with .npy or .npz, the file is loaded with `numpy.load`
+    (unless *asfileobj* is `False`).
```

Comment:
And `if` + `unless` is confusing.
```suggestion
    filename ends with .npy or .npz, and *asfileobj* is `True`, the file is loaded with `numpy.load`.
```
(+ re-wrap if needed)

Reply:
sure.